scripts/removeAnchors.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# This script removes all code anchors to improve readability
+
+# All .purs files in the exercises directories (excluding hidden files)
+ALL_PURS=$(find exercises \( ! -regex '.*/\..*' \) -type f -name '*.purs')
```

Comment:
This works on linux, but we should also get an osx user to test this so we don't repeat another #194.

Reply:
Confirmed this works on osx too